src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(operand) {
+      this.result += operand;
+
+      return this;
```

Comment:
you don't need to return this from add subtract multiply and divide methods. cinse only operate and reset methods could be chained

Reply:
done